roles/ceph-osd/tasks/scenarios/collocated.yml

```diff
@@ -57,4 +73,17 @@
     - "{{ devices }}"
   when:
     - not containerized_deployment
-    - item.0.partitions|length == 0
\ No newline at end of file
+  register: osd
+
+- name: set_fact osd_created
+  set_fact:
+    osd_created: True
+  when: osd.changed
+
+- name: fail if OSD was never created
```

Comment:
oops missed this one, no uppercase in `name`'s task.

Reply:
@leseb done.  Out of curiosity is that an ansible standards type of thing?  Or is that just something decided on for this particular project?

Reply:
This is something we decided for this particular project.